### PR TITLE
Cloud Native Chapter 2

### PR DESCRIPTION
**Pull request by ardalis**
## Summary

Chapter 2 Files - Images are already in the ch1 PR.

### CODE REVIEW

File: docs/architecture/cloud-native/leverage-serverless-functions.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+In the spectrum of managing full machines and operating systems to leveraging cloud capabilities, serverless lives at the extreme end where the only thing you're responsible for is your code, and you only pay for your code when it runs. Azure Functions provide a way to build serverless capabilities into your applications. Traditionally, Azure Functions were built on the Azure platform or on infrastructure running the Azure Stack or the Azure Functions Runtime. However, you can also run Azure Functions within a Docker container for additional portability and flexibility. Adding support for docker to an Azure Functions project is as simple as adding a switch to the command line:
+
+```cli
+func init ProjectName --docker
+```
+
+When creating an Azure Function, you must choose the programming language you want to use from these options:
+
+- dotnet (C#)
```
Comment (mairaw) [inline docs/architecture/cloud-native/leverage-serverless-functions.md]: is this the actual values that you need to input somewhere?

**Comment by mairaw** [comment]
@ardalis the review for this chapter is done. Please take a look at the changes I've made to see if you're ok with them and at my question. If those are values, we need to prevent them from being localized. If you're talking about the languages in general sense, then you'd need to use the proper name: .NET, Python, etc.

File: docs/architecture/cloud-native/leverage-serverless-functions.md (hunk 1 above)
Comment (ardalis) [reply docs/architecture/cloud-native/leverage-serverless-functions.md]: Yes, there's a command line dialog and you choose from the options listed.

**Comment by ardalis** [comment]
@mairaw Answered. How do we prevent them being localized? Put them like `dotnet` or something?

**Comment by ardalis** [comment]
Your updates are good, thanks! :shipit:

**Comment by mairaw** [comment]
Correct, backticks prevent them from being localized because they'll be considered "code"

File: docs/architecture/cloud-native/leverage-serverless-functions.md (hunk 1 above)
Comment (ardalis) [reply docs/architecture/cloud-native/leverage-serverless-functions.md]: Fixed.

**Review by ardalis — COMMENTED** [review]
Fixed backticks

**Review by mairaw — APPROVED** [review]
Awesome! Let's merge.

**Pull request merged**